dbms/src/Processors/Executors/PipelineExecutor.cpp

```diff
@@ -11,6 +11,8 @@
 #include <Processors/ISource.h>
 #include <Common/setThreadName.h>
 
+#include <sched.h>
```

Comment:
Looks like this also can be omitted for apple/freebsd

Reply:
Yes.
Actually, it's better to add affinity for other os too.